build_your_own_images.md

```diff
@@ -0,0 +1,185 @@
+# Building Docker images that contain the Kong Gateway
+
+Kong Software uses the [docker-kong github
+repository](https://github.com/Kong/docker-kong/) to build Docker images that
+contain the Kong Gateway. We will use  
+[docker-kong](https://github.com/Kong/docker-kong/) as a reference
+implementation for describing how to build your own Docker images that contain the Kong
+Gateway.
+
+To build your Docker image, you will need to provide 
```

Comment:
I picked this approach as a way of giving people a tour of our Dockerfiles as preparation for writing their own.